docs/usage_general.rst.inc

```diff
@@ -249,22 +249,31 @@ Some automatic "answerers" (if set, they automatically answer confirmation quest
     answer or ask you interactively, depending on whether retries are allowed (they by default are
     allowed). So please test your scripts interactively before making them a non-interactive script.
 
+.. _XDG env var: https://specifications.freedesktop.org/basedir-spec/0.6/ar01s03.html
+
 Directories and files:
     BORG_BASE_DIR
-        Default to '$HOME', '~$USER', '~' (in that order)'.
-        If we refer to ~ below, we in fact mean BORG_BASE_DIR.
+        Defaults to ``$HOME`` or ``~$USER`` or ``~`` (in that order).
+        If you want to move all borg-specific folders to a custom path at once, all you need to do is
+        to modify ``BORG_BASE_DIR``: the other paths for cache, config etc. will adapt accordingly.
+        (Assuming you didn't set them to a different custom value.)
     BORG_CACHE_DIR
-        Default to '~/.cache/borg'. This directory contains the local cache and might need a lot
+        Defaults to ``BORG_BASE_DIR/.cache/borg``. If ``BORG_BASE_DIR`` is not explicitly set while
```

Comment:
as we talk about $HOME, guess we need $BORG_BASE_DIR here when talking about pathes.

Reply:
same at other misc. places, please fix all places within a path to use `$BORG.../...` or `$XDG.../...`.

Reply:
:+1:

Reply:
ok, solved.